client/ayon_houdini/api/plugin.py

```diff
@@ -24,6 +24,41 @@
 
 SETTINGS_CATEGORY = "houdini"
 
+REMAP_CREATOR_IDENTIFIERS: Dict[str, str] = {
```

Comment:
NOTE: With these changes you can't open new scenes with older addon versions.

Suggestion: keep `openpype` identifier in create plugins for now. Change this remaping to convert `ayon` identifiers to `openpype` identifiers. After at about month change the create plugin identifiers and revert this mapping.

Reply:
Implemented with https://github.com/ynput/ayon-houdini/pull/245/commits/e31fe40549e6bd17273d22f3916b1c72a1be4a0a

Reply:
I would also bump minor with this PR. And bump major when it is completelly finished.

Reply:
Added "bump minor" label.